AntennaTracker/tracking.cpp

```diff
@@ -31,10 +31,17 @@ void Tracker::update_vehicle_pos_estimate()
  */
 void Tracker::update_tracker_position()
 {
+    struct Location temp_loc = current_loc;
+
     // update our position if we have at least a 2D fix
     // REVISIT: what if we lose lock during a mission and the antenna is moving?
-    if (!ahrs.get_position(current_loc) && (gps.status() >= AP_GPS::GPS_OK_FIX_2D)) {
+    if (ahrs.get_position(current_loc)) {
+        stationary = false;
+    } else if (gps.status() >= AP_GPS::GPS_OK_FIX_2D) {
```

Comment:
I think we should eliminate these gps cases; we have an AHRS for a reason and we should trust it if it thinks the position isn't viable.  It does lots of additional checks, most notable velocity dop checks.

Reply:
fixed